content/en/docs/Tutorials/multicluster/09-Configure-Kiali.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Configure Kiali for multicluster"
+description: "In this section we will add some configuration for Kiali to start observing the remote cluster."
+weight: 9
+---
+
+We will configure Kiali to access the remote cluster. This will require a secret (similar to the Istio secret) containing the credentials for Kiali to fetch information for the remote cluster:
+
+```
+curl -L -o kiali-prepare-remote-cluster.sh https://raw.githubusercontent.com/kiali/kiali/master/hack/istio/multicluster/kiali-prepare-remote-cluster.sh
+
+chmod +x kiali-prepare-remote-cluster.sh
+
+./kiali-prepare-remote-cluster.sh --kiali-cluster-context $CLUSTER_EAST --remote-cluster-context $CLUSTER_WEST
+```
+
+Finally, upgrade the installation for Kiali to pick up the secret:
+
+```
+helm upgrade \ 
```

Comment:
I had some issues in this line: 

`Error: "helm upgrade" requires 2 arguments`

Probably an indentation error, but if I put this in the same line: 

`Error: UPGRADE FAILED: "kiali-server" has no deployed releases`

I guess that should be in `istio-system` namespace, in this case I get: 

`Error: UPGRADE FAILED: cannot patch "kiali" with kind ClusterRoleBinding: ClusterRoleBinding.rbac.authorization.k8s.io "kiali" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"kiali-viewer"}: cannot change roleRef`

Reply:
I will check this, it's rare because I tested this but I will double check.